augur/api/view/utils.py

```diff
@@ -298,3 +308,68 @@ def render_message(messageTitle, messageBody = None, title = None, redirect = No
 def render_module(module, **args):
     args.setdefault("body", module)
     return render_template('index.j2', **args)
+
+""" ----------------------------------------------------------------
+    No longer used
+"""
+# My attempt at a loading page
+def renderLoading(dest, query, request):
+    cache_files_requested.append(request)
+    return render_template('index.j2', body="loading", title="Loading", d=dest, query_key=query, api_url=getSetting('serving'))
+
+with DatabaseEngine() as engine:
```

Comment:
I think it's fine to merge for testing, but before this makes it into main, we need to ensure that the repo load controller gets updated with the below syntax (or something similar). This is just here for testing, and is not intended to be final